students/marc_charbo/assignment_1/generator_solution.py

```diff
@@ -0,0 +1,35 @@
+def doubler(max):
+    x, y = 2, 0
+    while y < max:
+        yield x ** y
+        y += 1
+
+def intsum(max):
```

Comment:
So the idea behind a generator function is you don't need to define a max, as you are iterating through it like a for loop where it will just run forever.  The use of `next()` just calls the next item.